rfcs/text/0001-wildcard-data-type.md

```diff
@@ -21,29 +28,10 @@ For a field to use wildcard, it will require changing the field's defined schema
 
 | Field Set | Field(s) |
 | --------- | -------- |
-| [`agent`](0001/agent.yml) | `agent.build.original` |
-| [`as`](0001/as.yml) | `as.organization.name` |
-| [`client`](0001/client.yml) | `client.domain`<br> `client.registered_domain` |
-| [`destination`](0001/destination.yml) | `destination.domain`<br> `destination.registered_domain` |
-| [`dns`](0001/dns.yml) | `dns.question.name`<br> `dns.answers.data` |
-| [`error`](0001/error.yml) | `error.stack_trace`<br> `error.type` |
-| [`file`](0001/file.yml) | `file.directory`<br> `file.path`<br> `file.target_path` |
-| [`geo`](0001/geo.yml) | `geo.name` |
-| [`host`](0001/host.yml) | `host.hostname`<br> |
-| [`http`](0001/http.yml) | `http.request.referrer`<br> `http.request.body.content`<br> `http.response.body.content` |
-| [`log`](0001/log.yml) | `log.file.path`<br> `log.logger` |
-| [`organization`](0001/organization.yml) | `organization.name` |
-| [`os`](0001/os.yml) | `os.name`<br> `os.full` |
-| [`pe`](0001/pe.yml) | `pe.original_file_name` |
-| [`process`](0001/process.yml) | `process.command_line`<br> `process.executable`<br> `process.name`<br> `process.thread.name`<br> `process.title`<br> `process.working_directory`<br> |
-| [`registry`](0001/registry.yml) | `registry.key`<br> `registry.path`<br> `registry.data.strings` |
-| [`server`](0001/server.yml) | `server.domain`<br> `server.registered_domain` |
-| [`source`](0001/source.yml) | `source.domain`<br> `source.registered_domain` |
-| [`tls`](0001/tls.yml) | `tls.client.issuer`<br> `tls.client.subject`<br> `tls.server.issuer`<br> `tls.server.subject` |
-| [`url`](0001/url.yml) | `url.full`<br> `url.original`<br> `url.path`<br> `url.domain`<br> `url.registered_domain` |
-| [`user`](0001/user.yml) | `user.name`<br> `user.full_name`<br> `user.email` |
-| [`user_agent`](0001/user_agent.yml) | `user_agent.original` |
-| [`x509`](0001/x509.yml) | `x509.issuer.distinguished_name`<br> `x509.subject.distinguished_name` |
+| [`error`](0001/error.yml) | `error.stack_trace` |
+| [`http`](0001/http.yml) | `http.request.body.content`<br> `http.response.body.content` |
+| [`process`](0001/process.yml) | `process.command_line` |
```

Comment:
If we could also keep `file.path` and `process.executable`, I think that would be valuable. I think it's okay to ignore `process.parent.executable` though.

Reply:
A large area of feedback from benchmarking was file paths would likely compress very well as `keyword` since file paths share prefixes. Even fields with moderate cardinality (50,000 - 100,000 unique values) saw increases of 20-30x storage space 😞 

Doesn't mean that all fields that capture file paths are out of consideration for `wildcard`; we just need to weigh any gains against the (fairly steep) costs.